Assets/VRTK/Prefabs/Internal/Scripts/VRTK_RadialMenu.cs

```diff
@@ -85,27 +100,27 @@ public enum ButtonEvent
         /// The HoverButton method is used to set the button hover at a given angle.
         /// </summary>
         /// <param name="angle">The angle on the radial menu.</param>
-        public virtual void HoverButton(float angle)
+        public virtual void HoverButton(TouchAngleDeflection tad)
```

Comment:
This (and in the other public accessors) makes changes to an existing public method that will cause breaking changes for anyone upgrading.

The correct solution would be to provide a new method and deprecate this method and then in this deprecated method just build up the appropriate call to the new method.

This way people will get the deprecation warning and know they need to upgrade their code, but things won't immediately break.

Reply:
I've [made this change](https://github.com/Fulby/VRTK/commit/4836289b8c51cbcd4cc38f060e26a32c358633eb) but any code which overrode HoverButton(float) will still break as the radial controller no longer calls that method. I'm not sure how to fix that?

Reply:
If people have override it, then that's up to them to fix :)

I'd suggest changing the deprecation message to match the format of existing deprecation messages:

"`ClassName.methodName` has been replaced with `ClassName.methodName`. This method will be removed in a future version of VRTK."

Reply:
OK cool. I've fixed the deprecation messages.